ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -0,0 +1,85 @@
+.pod-logs {
+  height: 90%;
```

Comment:
Do you mean a vertical scroll on the whole panel, or just the logs viewer? If the former, that's because of the class `application-details__tab-content-full-height`. I can't reproduce a vertical scroll for the actual logs viewer

Reply:
Here it is:

https://user-images.githubusercontent.com/426437/105561294-2d1e2980-5ccb-11eb-9c7a-a1b1eb32e8e9.mov

Events tab don't have a scroll but logs tab has it.

Reply:
Tested current version of log viewer behavior - it has it too, so it is old bug. Feel free to ignore this comment :)